python/core/auto_generated/mesh/qgsmeshrenderersettings.sip.in

```diff
@@ -144,7 +145,7 @@ values on vertices
 .. versionadded:: 3.12
 %End
 
-    void setDataInterpolationMethod( const DataInterpolationMethod &dataInterpolationMethod );
+    void setDataResamplingMethod( const DataResamplingMethod &dataResamplingMethod );
```

Comment:
This is an API break, it's already part of the stable API...

Reply:
Yes true, but the old name is not very exact anymore. Does the mention in the doc "\note The API is considered EXPERIMENTAL and can be changed without a notice" is not sufficient to allows API break ?
Or maybe deprecate the old implementation ?

Reply:
That's usually ok then. When do you expect this to stabilise? At some stage I want to add automatic conversion of ESRI TIN styling to Mesh layers into SLYR, and would need a stable API for that

Reply:
I understand... I will talk about with @PeterPetrik

Reply:
we are close to stable API now with 1D, 2D and 3D ready in meshes. I think after moving to the new time management, it should be mostly it.

Reply:
Well.. then I can definitely wait :smile: